### ISSUE TITLE
Parsing Error

### ISSUE THREAD

**Issue by patrickoleary1977** [nebulous/infinitude#93]
I am getting the following error.  I'm a noob, but did go through and make sure I had all of the prerequisites.  Thoughts?  I'm on Mac BTW...

Can't locate Data/ParseBinary.pm in @INC (you may need to install the Data::ParseBinary module) (@INC contains: /Users/patrick/Desktop/infinitude-master/lib /Library/Perl/5.18/darwin-thread-multi-2level /Library/Perl/5.18 /Network/Library/Perl/5.18/darwin-thread-multi-2level /Network/Library/Perl/5.18 /Library/Perl/Updates/5.18.4/darwin-thread-multi-2level /Library/Perl/Updates/5.18.4 /System/Library/Perl/5.18/darwin-thread-multi-2level /System/Library/Perl/5.18 /System/Library/Perl/Extras/5.18/darwin-thread-multi-2level /System/Library/Perl/Extras/5.18 .) at /Users/patrick/Desktop/infinitude-master/lib/CarBus/Frame.pm line 4.
BEGIN failed--compilation aborted at /Users/patrick/Desktop/infinitude-master/lib/CarBus/Frame.pm line 4.
Compilation failed in require at /Users/patrick/Desktop/infinitude-master/lib/CarBus.pm line 3.
BEGIN failed--compilation aborted at /Users/patrick/Desktop/infinitude-master/lib/CarBus.pm line 3.
Compilation failed in require at ./infinitude line 49.

**Comment by nebulous** [comment]
That looks like you'd just need to `cpanm Data::ParseBinary` though if that module is missing I expect many others would be missing as well. How did you go about installing the prerequisites?

**Comment by patrickoleary1977** [comment]
I tried that, and it would not let me install it.

Whatever I did though, it boogered up my Homebridge setup, so I restored from a pre-infinitude clone.  The reality is, I know nothing about this stuff and basically find guides to walk me through installs.  I found a reddit guide that was working well enough until I got stuck at the error I posted above. It seemed that the install deps . was not working correctly.
